src/explore-education-statistics-admin/src/modules/find-statistics/components/KeyStatisticsSecondary.tsx

```diff
@@ -104,8 +66,22 @@ export const AddSecondaryStats = ({
         )}
 
         <ModalConfirm
-          onConfirm={async () => {
-            await removeSecondarySectionBlock();
+          onConfirm={() => {
+            if (release.keyStatisticsSecondarySection?.content) {
+              release.keyStatisticsSecondarySection.content.forEach(
```

Comment:
Honestly, this section should only be allowed to have one content block, but that's only limited by the UI.. (not in the API)
so instead of doing a delete of `section.content[0]`, we do a delete request for all the content items.. just incase somehow 2 or more blocks got added

Reply:
Added the Promise.all